main.go

```diff
@@ -114,20 +115,29 @@ func main() {
 			if (*flagMaxFails > 0) && (fails >= *flagMaxFails) {
 				atomic.StoreInt32(&exitFlag, 1)
 			}
-			f, err := ioutil.TempFile("", "go-stress")
-			if err != nil {
-				fmt.Printf("failed to create temp file: %v\n", err)
-				os.Exit(1)
-			}
-			f.Write(out)
-			f.Close()
-			if len(out) > 2<<10 {
-				out = out[:2<<10]
+			if *flagStdErr {
+				fmt.Fprintf(os.Stderr, "\n%s\n", out)
```

Comment:
I tend to see stderr as the "verbose" output, which is definitely true in this case.
keeping info messages on stdout, especially the final success status, also seems more appropriate.
ultimately, it's more about synchronizing this tool with whatever consumes its output.